### PR TITLE
improve(polardb): feature polardb support account

### PR DESCRIPTION
**Pull request by zhangyuangang**
3-account

### CODE REVIEW

File: alicloud/resource_alicloud_polardb_account.go [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+	// wait instance running before modifying
+	if err := polarDBService.WaitForPolarDBInstance(request.DBClusterId, Running, DefaultTimeoutMedium); err != nil {
+		return WrapError(err)
+	}
+	err := resource.Retry(5*time.Minute, func() *resource.RetryError {
+		raw, err := client.WithPolarDBClient(func(polarDBClient *polardb.Client) (interface{}, error) {
+			return polarDBClient.CreateAccount(request)
+		})
+		if err != nil {
+			if IsExceptedErrors(err, OperationDeniedDBStatus) {
```
Comment (Lexsss) [inline alicloud/resource_alicloud_polardb_account.go]: I think it's not needed

File: alicloud/resource_alicloud_polardb_account.go [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+	}
+	request := polardb.CreateDeleteAccountRequest()
+	request.RegionId = client.RegionId
+	request.DBClusterId = parts[0]
+	request.AccountName = parts[1]
+
+	raw, err := client.WithPolarDBClient(func(polarDBClient *polardb.Client) (interface{}, error) {
+		return polarDBClient.DeleteAccount(request)
+	})
+	if err != nil && !IsExceptedError(err, InvalidAccountNameNotFound) {
```
Comment (Lexsss) [inline alicloud/resource_alicloud_polardb_account.go]: I think InvalidAccountNameNotFound should return nil

File: alicloud/resource_alicloud_polardb_account_test.go [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+
+		Providers:    testAccProviders,
+		CheckDestroy: rac.checkResourceDestroy(),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccConfig(map[string]interface{}{
+					"db_cluster_id":    "${alicloud_polardb_cluster.cluster.id}",
+					"account_name":     "tftestnormal",
+					"account_password": "YourPassword_123",
+					"account_type":     string("Normal"),
```
Comment (Lexsss) [inline alicloud/resource_alicloud_polardb_account_test.go]: "Normal"

File: alicloud/resource_alicloud_polardb_account_test.go [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+		default = "8.0"
+	}
+
+	variable "instanceclass" {
+		default = "polar.mysql.x4.large"
+	}
+
+	resource "alicloud_polardb_cluster" "cluster" {
+		db_type = "${var.engine}"
+		db_version = "${var.engineversion}"
```
Comment (Lexsss) [inline alicloud/resource_alicloud_polardb_account_test.go]: Please remove the variables and write them directly in the resource

File: alicloud/resource_alicloud_polardb_account.go (hunk 2 above)
Comment (zhangyuangang) [reply alicloud/resource_alicloud_polardb_account.go]: > I think InvalidAccountNameNotFound should return nil

done

File: alicloud/resource_alicloud_polardb_account_test.go (hunk 3 above)
Comment (zhangyuangang) [reply alicloud/resource_alicloud_polardb_account_test.go]: > "Normal"

done

File: alicloud/resource_alicloud_polardb_account.go (hunk 1 above)
Comment (zhangyuangang) [reply alicloud/resource_alicloud_polardb_account.go]: > I think it's not needed

I think it`s ok

File: alicloud/resource_alicloud_polardb_account_test.go (hunk 4 above)
Comment (zhangyuangang) [reply alicloud/resource_alicloud_polardb_account_test.go]: > Please remove the variables and write them directly in the resource

done

File: alicloud/resource_alicloud_polardb_account.go [hunk 5]
```diff
@@ -0,0 +1,206 @@
...
+				Required: true,
+			},
+
+			"account_name": {
+				Type:     schema.TypeString,
+				ForceNew: true,
+				Required: true,
+			},
+
+			"account_password": {
```
Comment (xiaozhu36) [inline alicloud/resource_alicloud_polardb_account.go]: please support kms_password to set a encrypted password. You can refer to alicloud_db_account or alicloud_instance

File: alicloud/resource_alicloud_polardb_account.go [hunk 6]
```diff
@@ -0,0 +1,206 @@
...
+
+	// Description will not be set when account type is normal and it is a API bug
+	if v, ok := d.GetOk("account_description"); ok && v.(string) != "" {
+		request.AccountDescription = v.(string)
+	}
+	// wait instance running before modifying
+	if err := polarDBService.WaitForPolarDBInstance(request.DBClusterId, Running, DefaultTimeoutMedium); err != nil {
+		return WrapError(err)
+	}
+	err := resource.Retry(5*time.Minute, func() *resource.RetryError {
```
Comment (xiaozhu36) [inline alicloud/resource_alicloud_polardb_account.go]: please remove the line 75-77 and they are redundant with the re-try block.

File: alicloud/resource_alicloud_polardb_account.go [hunk 7]
```diff
@@ -0,0 +1,206 @@
...
+		raw, err := client.WithPolarDBClient(func(polarDBClient *polardb.Client) (interface{}, error) {
+			return polarDBClient.CreateAccount(request)
+		})
+		if err != nil {
+			if IsExceptedErrors(err, OperationDeniedDBStatus) {
+				return resource.RetryableError(err)
```
Comment (xiaozhu36) [inline alicloud/resource_alicloud_polardb_account.go]: Please add 5 second interval for the retry.

**Pull request merged**